scripts/_download.js

```diff
@@ -180,26 +199,201 @@ export async function downloadAndExtractTarGz({ targetDirectory, url, sha256 })
 
     // If a sha256 is provided, verify against the downloaded file
     console.log(`Asset sha256: ${green(downloadSha256)}`);
-    if (sha256) {
-      if (downloadSha256 === sha256) {
-        console.log(`${green("Verified!")} Asset's sha256 matches expected`);
-      } else {
-        throw new Error(
-          `Downloaded file ${targetFile} sha256 ${downloadSha256} does not match the expected sha256 ${sha256}`,
-        );
-      }
+    if (sha256 && downloadSha256 !== sha256) {
+      throw new Error(
+        `Downloaded file ${targetFile} sha256 ${downloadSha256} does not match the expected sha256 ${sha256}`,
+      );
```

Comment:
Doesn't make sense to me to remove the "Verified" output statment

Reply:
added it back